sources/packages/web/src/components/common/SearchStudentsCommon.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+  <full-page-container>
```

Comment:
This component can have just the search and result content and not the full page container.

They way how we follow is we create sepeate views for AEST and Institution at /views.

We use the Common component inside both the views like this.


![image](https://user-images.githubusercontent.com/54600590/230186978-edb813de-98b4-4c61-95c9-c61be7058379.png)